R/members.R

```diff
@@ -157,25 +157,27 @@ space_member_add.data.frame <- function(space, users, ...) {
 #' @inheritParams space_info
 #' @param users ID number or email of the user to be removed, or a data frame
 #'   with either a `user_id` or `email` column.
-#' @param remove_projects Whether to remove user's projects from the workspace
-#'   and move them to their personal space or to keep the projects in the
-#'   workspace.
+#' @param content_action What to do with the users content after they are
+#'   removed from the space. Options for the content are to: "leave"
+#'   leaving the content where it is, "archive" moving the content to
```

Comment:
Seeing "leave" on its own made me realize it might be a bit counter-intuitive -- sounds like leaving a place more than leaving in place. I wonder if the option could be "keep" instead?

- "keep": keep content where it is
- "archive": move tp space archive
- "trash": move to space trash

What do you think?

Reply:
I think those would make sense. I might re-map them here. It is a bit more difficult on my end to tweak them at this point.